Makefile

```diff
@@ -84,6 +86,10 @@ clean: $(SUBPACKAGES.CLEAN)
 
 tests: $(SUBPACKAGES) $(SUBPACKAGES.TESTS)
 
+run-tests: $(SUBPACKAGES.RUNTESTS)
```

Comment:
These both depend on having the `tests` target run, right?

Reply:
Indeed :P